.travis.yml

```diff
@@ -21,6 +25,6 @@ script:
 - if test x"$TOKENV" != x'py24' ; then  tox ; fi
 - if test x"$TOKENV" = x'py24' ; then python2.4 -V && python2.4 -m compileall -fq -x 'module_utils/(a10|rax|openstack|ec2|gce).py' lib/ansible/module_utils ; fi
```

Comment:
I know you didn't touch these 2 lines, but since you are in here.  Looks like `TOKENV` should probably be `TOXENV`